docs/docs-content/clusters/edge/site-deployment/deploy-custom-registries/deploy-external-registry.md

```diff
@@ -114,7 +114,18 @@ information, refer to [Enable Local Harbor Registry](./local-registry.md).
    [Upload Cluster Images to External Registry with Palette Edge CLI](./upload-images-to-registry.md) to learn how to
    use the Palette Edge CLI to upload all images in a cluster profile to an external registry.
 
-9. Follow the [Create Cluster Definition](../site-installation/cluster-deployment.md) guide and deploy your cluster.
+9. In the Kubernetes layer of your cluster profile, remove `AlwaysPullImages` from
+   `cluster.config.clusterConfiguration.apiServer.extraArgs.enable-admission-plugins` and add it to
+   `cluster.config.clusterConfiguration.apiServer.extraArgs.enable-admission-plugins`.
```

Comment:
@lennessyy We need to just remove `AlwaysPullImages` and don't have to add it anywhere. So we should remove "add it to
   `cluster.config.clusterConfiguration.apiServer.extraArgs.enable-admission-plugins`"